src/renderer/components/commands-address-bar.tsx

```diff
@@ -96,13 +121,85 @@ export class AddressBar extends React.Component<AddressBarProps, AddressBarState
     this.setState({ value: event.target.value });
   }
 
+  public async fetchExampleAndLoad(ref: string, path: string): Promise<boolean> {
+    const { appState } = this.props;
+    try {
+      const octo = await getOctokit();
+
+      // You can load Gists without being authenticated,
+      // but we get better rate limits when authenticated
+      if (appState.gitHubToken) {
+        octo.authenticate({
+          type: 'token',
+          token: appState.gitHubToken!
+        });
+      }
+
+      // TODO: Maybe we should fetch the package.json for the given ref to use the correct version of Electron?
+
+      const folder = await octo.repos.getContents({
```

Comment:
I wonder if we can even hit this endpoint more than once without being authenticated 🤔

Reply:
If we're ok with making more requests and relying on 404's I think we can make this work without using the API (just hitting the `raw.*` URLs and guessing for `main.js`, `renderer.js` and `index.html`